citing-us/index.html

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Citing OME
```

Comment:
The naming/menu title will ultimately be decided by Jason. We'll bring it up at the next meeting, and I'll change the files accordingly.

I'll work on the other edits below and push in a bit.